phoenix-core/src/it/java/org/apache/phoenix/calcite/CalciteIT.java

```diff
@@ -1343,6 +1343,28 @@ public void initTable() throws Exception {
                         {7L, "0000000005", "T5", "0000000005", "S5"},
                         {8L, "0000000006", "T6", "0000000006", "S6"}})
                 .close();
+
```

Comment:
Thank you for the info, @lomoree! Very helpful! I was wondering if there had been any check of that kind during conversion. Anyway, would you mind removing that NO_SEQUENCE predicate and trying the changes I suggested below?